tests/integration/test_bootstrap_interface.py

```diff
@@ -0,0 +1,172 @@
+import re
+import json
+import pytest
+from time import sleep
+
+# ----- bootstrap server ---------------------------------------------------------------------------------------------------------------
+# qleisan_wakaama git:(github-ci-integration-test_newer) ✗ build-wakaama/examples/bootstrap_server/bootstrap_server -f examples/bootstrap_server/bootstrap_server.ini
+# LWM2M Bootstrap Server now listening on port 5685.
+
+# > 26 bytes received from [::1]:56830
```

Comment:
What is the reason to have such a wall of text? Looks like something that outdates very fast and is of limited value.